### PR TITLE
Tdfranklin/ch1344/ch1345/update public api docs

### PR DESCRIPTION
**Pull request by tdfranklin**
There's probably still a few things that need to be cleaned up here and there and there's quite a few descriptions that probably need to be changed or added, but in general I think the layout is looking a lot better.  I look forward to hearing you guys feedback!

### CODE REVIEW

File: source/includes/_fabrics.md [hunk 1]
```diff
@@ -61,19 +61,19 @@ Standard Attributes
...
+| country_origin <br> <span>string</span>             | Where the fabric was milled                                        |
+| fabric_weight_grams_meter <br> <span>integer</span> | Weight in grams per square meter                                   |
+| fabric_grouping <br> <span>string</span>            | Fabrics are typically grouped by dominant color                    |
+| pattern <br> <span>string</span>                    | Type of pattern. Common values are `stripe`, `check`, and `solid`. |
+| last_stock_edit_date <br> <span>datetime</span>     | The last time the fabric inventory level was changed               |
+| fabric_year <br> <span>year</span>                  | The year the fabric was released                                   |
+| collection <br> <span>subresource</span>            | The collection in which the fabric was released                    |
+| composition <br> <span>subresource</span>           | The material composition of the fabric                             |
+| mill <br> <span>subresource</span>                  | *Description TBD*                                                  |
```
Comment (geori) [inline source/includes/_fabrics.md]: The mill that produced the fabric.

File: source/includes/_fabrics.md [hunk 2]
```diff
@@ -61,19 +61,19 @@ Standard Attributes
...
+| --------------------------------------------------- | ------------------------------------------------------------------ |
+| country_origin <br> <span>string</span>             | Where the fabric was milled                                        |
+| fabric_weight_grams_meter <br> <span>integer</span> | Weight in grams per square meter                                   |
+| fabric_grouping <br> <span>string</span>            | Fabrics are typically grouped by dominant color                    |
+| pattern <br> <span>string</span>                    | Type of pattern. Common values are `stripe`, `check`, and `solid`. |
+| last_stock_edit_date <br> <span>datetime</span>     | The last time the fabric inventory level was changed               |
+| fabric_year <br> <span>year</span>                  | The year the fabric was released                                   |
+| collection <br> <span>subresource</span>            | The collection in which the fabric was released                    |
+| composition <br> <span>subresource</span>           | The material composition of the fabric                             |
```
Comment (geori) [inline source/includes/_fabrics.md]: Lets add an example: `(E.g, 100% Wool)`

File: source/includes/_orders.md [hunk 3]
```diff
@@ -75,27 +95,27 @@ The Orders API returns order, garment, order status, and delay status objects.
...
+| ship_type <br> <span>string</span>           | How is the garment shipped to the final destination                                                                                                                                                                   |
+| ship_cost <br> <span>decimal</span>          | How much did it cost to ship the whole order                                                                                                                                                                          |
+| subtotal <br> <span>decimal</span>           | Order amount before tax and discounts are included                                                                                                                                                                    |
+| dealer_discount <br> <span>decimal</span>    | *Description TBD*                                                                                                                                                                                                     |
```
Comment (geori) [inline source/includes/_orders.md]: Found it: line 345 of pricing says: `dealer_discount is the amount of dealer's permanent discounts`

File: source/includes/_orders.md [hunk 4]
```diff
@@ -132,71 +152,207 @@ currency | subresource | currency used in the order
...
+| option_cost <br> <span>decimal</span>       | Dollar cost of the premium options used in this garment                                                          |
+| garment_type <br> <span>string</span>       | Type of garment. [Click here](#garment-types) for more info                                                      |
+| created_at <br> <span>datetime</span>       | When the garment was first created (but not ordered)                                                             |
+| updated_at <br> <span>datetime</span>       | When the garment was last modified                                                                               |
+| order_status <br> <span>subresource</span>  | Current status of the order. [Click here](#order-statuses) for more info                                         |
+| delay_status <br> <span>subresource</span>  | If garment is delayed, this will return why. [Click here](#delay-statuses) for more info                         |
+| dealer_order <br> <span>subresource</span>  | *Description TBD*                                                                                                |
```
Comment (geori) [inline source/includes/_orders.md]: The full order placed by a single customer.  Includes this garment and can include more garments.

File: source/includes/_orders.md [hunk 5]
```diff
@@ -208,13 +364,71 @@ updated_at | datetime | when the garment was last modified
...
-last_delay_change_date | datetime | when did the delay status last change to a new state
+| Attribute               | Type        | Description                                                                                                 |
+| ----------------------- | ----------- | ----------------------------------------------------------------------------------------------------------- |
+| prefix                  | string      | First part of the title. This indicates where the order was made or where the factory will send the garment |
+| index                   | string      | Sequence number (1/7, 2/7, 3/7 ...) for each garment in an order. Each item has a different index           |
+| last_status_change_date | datetime    | When did the order status last change to a new state                                                        |
+| last_delay_change_date  | datetime    | When did the delay status last change to a new state                                                        |
+| manufacturer            | subresource | Information about the manufacturer of the garment                                                           |
```
Comment (geori) [inline source/includes/_orders.md]: The factory who made the garment

File: source/includes/_orders.md [hunk 6]
```diff
@@ -208,13 +364,71 @@ updated_at | datetime | when the garment was last modified
...
+    "email": "info@trinity-apparel.com"
+}
+```
+
+Standard Attributes
+
+| Attribute                            | Description                       |
+| ------------------------------------ | --------------------------------- |
+| id <br> <span>integer</span>         | Unique identifier for the object  |
+| address_id <br> <span>integer</span> | *Description TBD*                 |
```
Comment (geori) [inline source/includes/_orders.md]: * We need to use a subresource for account

**Comment by tdfranklin** [comment]
@geori - OK - changes made.

**Pull request merged**